spec/models/rook_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe Rook, type: :model do
+  it 'has a valid Factory' do
+    build(:rook).expect be_valid
```

Comment:
Oops sorry the schema in #5 - https://github.com/teamdeepblue/caissa-chess/blob/c0e53e8ff3f3919c450cb17fdf50e1c2a2655aa0/db/schema.rb